src/Uno.SourceGeneratorTasks.Shared/Tasks/SourceGeneratorHostWrapper.cs

```diff
@@ -28,27 +28,37 @@
 using Uno.SourceGeneratorTasks.Helpers;
 using Uno.SourceGeneratorTasks.Logger;
 using System.Diagnostics;
+using System.Globalization;
 
 namespace Uno.SourceGeneratorTasks
 {
-    public class SourceGeneratorHostWrapper : MarshalByRefObject
+	public class SourceGeneratorHostWrapper : MarshalByRefObject
 	{
-        private readonly RemoteLoggerProvider _remoteLoggerProvider = new RemoteLoggerProvider();
+		private readonly RemoteLoggerProvider _remoteLoggerProvider = new RemoteLoggerProvider();
 		private bool _additionalAssembliesLoaded;
+		private string _mSBuildBasePath;
 
 		public SourceGeneratorHostWrapper()
-        {
-            LogExtensionPoint.AmbientLoggerFactory.AddProvider(_remoteLoggerProvider);
-            LogExtensionPoint.AmbientLoggerFactory.AddDebug();
+		{
+			LogExtensionPoint.AmbientLoggerFactory.AddProvider(_remoteLoggerProvider);
+			LogExtensionPoint.AmbientLoggerFactory.AddDebug();
+
+			ApplyVS4MacWorkarounds();
 
-            ApplyVS4MacWorkarounds();
+			RegisterAssmblyLoader();
```

Comment:
RegisterAss***e***mblyLoader ?

Reply:
Need to save space on disk, bytes are expensive ;-)